exporter/views.py

```diff
@@ -75,6 +75,9 @@ def do_bagit_export(assets, export_base_dir, export_filename_base):
         text_output_path = os.path.join(dest_path, "%s.txt" % asset_filename)
         with open(text_output_path, "w") as f:
             f.write(asset.latest_transcription or "")
+            if settings.ATTRIBUTION_TEXT:
```

Comment:
I'm wondering whether the setting location should be tweaked to make this opt-in rather than opt-out:

```suggestion
            if getattr(settings, "ATTRIBUTION_TEXT"):
```

… and then moving our value to `settings_prod.py`

Reply:
Yeah it should be this way.

Reply:
Please note that I left out the third getattr parameter – since it's not `dict.get` that should have been:

```suggestion
            if getattr(settings, "ATTRIBUTION_TEXT", None):
```

Reply:
I used hasattr instead of getattr - since we're not saving the result, it seems more appropriate.